src/pytest_update_test_results/plugin.py

```diff
@@ -2,7 +2,7 @@
 from typing import Dict
 
 from _pytest.reports import TestReport
-from pytest_merge_xml.merge_xml import modify_xml
+from pytest_update_test_results.update_test_results import modify_xml
 
 
 class XMLReportMergerPlugin:
```

Comment:
```suggestion
class UpdateTestResultsPlugin:
```

Reply:
Yes, --update-xml will be more coherent now. I'll update along with the docstring format change.